rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
```

Comment:
There's a corner case here to be tested and considered. What happens when no app is in `sync_status="Synced"`. I'm not sure the LHS of the operation will return 0 - it might not return anything. We need to have it tested and in case we have a problem there, we need to decide if we want to address it.

Reply:
I added the test, and it passed.

Reply:
The test you need to add is for `GitOpsApplicationSyncErrors` with no apps in `sync_status="Synced`
This test would probably fail because:
`sum(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"})` with values like: 
```
- series: 'argocd_app_info{sync_status="Synced",namespace="gitops-service-argocd"}'
        values: '1 1 1 1 1 1'
```
Will return `No datapoints found` and we will not be able to divide it with the next expression:
`sum(argocd_app_info{namespace="gitops-service-argocd"})`